src/utils/changeChain.ts

```diff
@@ -0,0 +1,69 @@
+import {
+  internalAddressesPerNetwork,
+  ValidNetworkNameTypes,
+  ChainAddresses,
+} from "../utils/contracts/contractAddresses/internalAddresses";
+import { injectedConnector } from "../hooks/injectedConnectors";
+
+declare class RequestParameters {
```

Comment:
`declare` is for ambient typing contexts; `export` is more appropriate here, and keeps consistency with the structure of the rest of the project.

Instead of classes, use interfaces, which allow you to use any type that has the right members, without having to use the explicit constructor of that type. This makes it easier for compositional functional programming due to its similarity to row typing.

```suggestion
export interface RequestParameters {
```